server/audit_math/bravo.py

```diff
@@ -251,7 +251,7 @@ def expected_prob(
     if threshold <= 0:
         return 0
 
-    n = asn
+    n = asn * p_w
```

Comment:
I noticed that #701 recommended multiplying by `p_wr` not `p_w` - was this intentional?

Reply:
ooooh good catch. Will fix.